src/ImageSharp/MetaData/ImageMetaData.cs

```diff
@@ -116,14 +118,12 @@ public double VerticalResolution
         /// </summary>
         public ExifProfile ExifProfile { get; set; }
 
-        /// <summary>
-        /// Gets or sets the frame delay for animated images.
-        /// If not 0, this field specifies the number of hundredths (1/100) of a second to
-        /// wait before continuing with the processing of the Data Stream.
-        /// The clock starts ticking immediately after the graphic is rendered.
-        /// </summary>
+        /// <inheritdoc/>
         public int FrameDelay { get; set; }
 
+        /// <inheritdoc/>
```

Comment:
Not sure why you want to use `inheritdoc` here but don't do it for the other fields. Think it's better to just put the comments here. Never seen `inheritdoc` do anything in the code completion.